gapic/batch/staging.yaml

```diff
@@ -0,0 +1,6 @@
+common:
+  batch_apis: '*'
+  batch_languages: '*'
+  api_config_pattern: ${THISDIR}/../api/artman_*.yaml
+  artman_language_yaml: ${THISDIR}/../lang/common.yaml
+  batch_output_dir: ${REPOROOT}/client-library-staging/generated
```

Comment:
I don't really think batch output should go to a different location from single language x api execution. Maybe output_dir should just be re-appropriated, and defaulted to the staging repo?